plugins/generator-1.18.2/forge-1.18.2/procedures/entity_set_stepheight.java.ftl

```diff
@@ -0,0 +1 @@
+${input$entity}.maxUpStep = (float) ${input$stepHeight};
```

Comment:
Use opt.toFloat system for casting, do not manually cast. Applies to all cases of this in the PR

Reply:
Can you send an example of this?

Reply:
Ctrl  + Shift + F in IDEA and enter "opt.toFloat", but sure:

![image](https://user-images.githubusercontent.com/16374228/222245526-0b4ee28e-caa8-4f13-a5ff-743e63b9d0cd.png)

Reply:
@kleiders3010 this part still needs work